src/classes/BDI_DataImport_API.cls

```diff
@@ -37,6 +37,11 @@
 global with sharing class BDI_DataImport_API {
 
      private class BDIException extends Exception {}
+    
+	public static boolean userCanEditSettings {get {
```

Comment:
@cdcarter this isn't needed in this in this class, right? It's only used in the controller?